src/main/java/io/vertx/codegen/CodeGenProcessor.java

```diff
@@ -37,7 +39,10 @@
 @javax.annotation.processing.SupportedSourceVersion(javax.lang.model.SourceVersion.RELEASE_8)
 public class CodeGenProcessor extends AbstractProcessor {
 
+  private static final String COULD_NOT_LOAD_JSON_MAPPERS_PROPERTIES = "Could not load json-mappers.properties";
+  private static final String JAVA_EXT = ".java";
```

Comment:
please no :-)

Reply:
Well, here we are so acculturated in do what SonarLint plugin asks for. But it is my bad do change things that are not the goal.
In this case, it was complaining about the number of times (> 3) that the string `.java` was used and it asks for changing them for a constant :)

Reply:
I'm not a sonar fan :-) , using ".java" is very ok and in fact I better understand ".java" string than JAVA_EXT